### PR TITLE
Task/cce 65 add cce as deployer option

### PR DESCRIPTION
**Pull request by dbrown-va**
JIRA Issue: https://vasdvp.atlassian.net/browse/CCE-65

Added Community Care to Product list and corresponding yaml + conf.

### CODE REVIEW

File: products/community-care.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DU_ARTIFACT=health-apis-community-care-deployment
+DU_VERSION=1.0.1
+DU_NAMESPACE=cce
+DU_DECRYPTION_KEY="$CRYPTO_KEY"
+DU_HEALTH_CHECK_PATH="/community-care/v0/eligibility/actuator/health"
```
Comment (jblefkove-va) [inline products/community-care.conf]: Should the version be in the path for the health check?

Comment (jhulbert-va) [reply products/community-care.conf]: This should be the exact path of your health check after the base url.

File: Jenkinsfile [hunk 2]
```diff
@@ -85,7 +85,7 @@ pipeline {
   }
   parameters {
     booleanParam(name: 'DEBUG', defaultValue: false, description: "Enable debugging output")
-    choice(name: 'PRODUCT', choices: ['none','data-query','exemplar','gal','squares','hotline','urgent-care','mock-ee'], description: "Install this product")
+    choice(name: 'PRODUCT', choices: ['none','data-query','exemplar','gal','squares','hotline','urgent-care','mock-ee','community-care'], description: "Install this product")
```
Comment (jhulbert-va) [inline Jenkinsfile]: Could you guys re-alphabetize these? I think that will make it easier to find products when deploying.

Comment (dbrown-va) [reply Jenkinsfile]: Sure, just pushed the update.

File: products/community-care.conf (hunk 1 above)
Comment (dbrown-va) [reply products/community-care.conf]: We are moving the `/v0/eligibility` portion of the path to the CCE controller, the health check will change shortly with `DU_VERSION=1.0.2`, to `/community-care/actuator/health`.

And the LB & Ingress will be only `/community-care/*` & `/community-care/(.*)` respectively.

**Pull request merged**